draft-ietf-rats-architecture.md

```diff
@@ -851,7 +862,7 @@ An Attestation Result that indicates compliance can be used by a Relying Party t
 authorization decisions based on the Relying Party's Appraisal Policy.  The simplest such
 policy might be to simply authorize any party supplying a compliant Attestation Result
 signed by a trusted Verifier.  A more complex policy might also entail comparing information
-provided in the result against known-good reference values, or applying more complex logic
+provided in the result against known-good Reference Values, or applying more complex logic
```

Comment:
Remove "known-good", assuming "known-good" is mentioned as an AKA for Reference Values.

Reply:
Done